gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidHandler.java

```diff
@@ -289,7 +281,7 @@ private void computeTimestampForExportCorner(TTimestampWrapper timestampWrapper,
                 if (cseValidRaoValidator.isSecure(raoResponse)) {
                     tcDocumentTypeWriter.fillTimestampExportCornerSuccess(timestamp, timestampWrapper.getMiecValue());
                 } else {
-                    runDichotomyForExportCorner(timestampWrapper, cseValidRequest, tcDocumentTypeWriter, jsonCracUrl, raoParametersURL);
+                    runDichotomyForExportCorner(timestampWrapper, cseValidRequest, tcDocumentTypeWriter, jsonCracUrl, raoParametersURL, network, cracCreationContext);
```

Comment:
So you pass the shifted network to the dichotomy, while we are still waiting for an answer on this subject